tests/time_tests/src/timetests/CMakeLists.txt

```diff
@@ -9,11 +9,13 @@ add_custom_target(time_tests)
 # Test target name is source file name without extension.
 FILE(GLOB tests "*.cpp")
 
+add_subdirectory("${OpenVINO_SOURCE_DIR}/tests/lib" tests_shared_lib)
```

Comment:
where `tests_shared_lib` folder will be placed? Should it be replaced with `${CMAKE_CURRENT_BINARY_DIR}/tests_shared_lib`?

Reply:
It's not needed due to `add_subdirectory` create folder in current build directory.